.travis.yml

```diff
@@ -1,12 +0,0 @@
-dist: trusty
-before_install:
-  - npm install -g npm@latest
-  - npm set progress=false
-language: node_js
-node_js:
-  - "8.16"
-after_success:
-  - npm run semantic-release
-branches:
-  except:
-    - "/^v\\d+\\.\\d+\\.\\d+$/"
```

Comment:
Any idea what this pattern is for?

Reply:
No idea, but as far as I understand [this](https://github.com/contentful/contentful-sdk-jsdoc/blob/fc78da763069a2fde8d3816618bf9bc178ead13a/package.json#L35) should make sure only `master` triggers a release.